src/main/java/org/mariadb/jdbc/internal/stream/PacketOutputStream.java

```diff
@@ -1,617 +1,665 @@
 package org.mariadb.jdbc.internal.stream;
 
-import java.io.*;
-import java.nio.ByteBuffer;
-import java.nio.ByteOrder;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.Reader;
+import java.io.UnsupportedEncodingException;
 import java.sql.Timestamp;
 import java.util.Calendar;
 import java.util.zip.DeflaterOutputStream;
 
+import org.mariadb.jdbc.internal.util.ByteBuffer;
 
 public class PacketOutputStream extends OutputStream {
```

Comment:
In my opinion these spacing changes make it VERY hard to evaluate this commit.  If you are going to do spacing changes (though not sure why you would in this case), you should probably do it in a separate commit.